setup.py

```diff
@@ -220,6 +220,7 @@ def run(self):
         "dmypy": "psutil >= 4.0",
         "python2": "typed_ast >= 1.4.0, < 2",
         "reports": "lxml",
+        "types": "pip",
```

Comment:
```suggestion
        "install-types": "pip",
```

Reply:
Yeah, that's probably correct :sweat_smile: